react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -181,6 +181,38 @@ if (buildGutenbergMobileJSBundle) {
     def bundleName = 'index.android.bundle'
     def jsRootDir = '../..'
 
+    // FIXME also check that submodule is properly up to date???
+    task abortIfGutenbergMobileSubmoduleNotClean(type: Exec) {
+        workingDir "../../"
+        commandLine "git", "status", "--porcelain"
+        standardOutput = new ByteArrayOutputStream()
+
+        // Storing this because we cannot access `standardOutput` in the doLast block
+        def output = { standardOutput.toString() }
+        doLast {
+            if (!output().isEmpty()) {
+                def message = "Attempting to build Gutenberg JS Bundle without a clean submodule. \
+Either clean the `libs/gutenberg-mobile` submodule (you probably want to do this) or \
+build Gutenberg from source by setting wp.BUILD_GUTENBERG_FROM_SOURCE to true."
+                throw new GradleException(message)
+            }
+        }
+    }
+
+    task removeGutenbergMobileSubmoduleChanges {
+        doLast {
+            println "Removing any changes in gutenberg-mobile project"
+            exec {
+                workingDir "../../"
```

Comment:
Got to admit, I'm feeling uncomfortable having a build script undo local changes to source folder. I understand though that our goal here is to clean up the `bundle` subfolder so, what do you think about specifying directly that folder to clean up? (Same for the `git clean -f` command.